tensorflow_datasets/core/visualization.py

```diff
@@ -66,17 +70,13 @@ def show_examples(ds_info, ds, rows=3, cols=3, plot_scale=3., image_key=None):
   plt = lazy_imports_lib.lazy_imports.matplotlib.pyplot
 
   if not image_key:
```

Comment:
Also use `if not image_key and not audio_key` after adding audio key param and generate audio_keys using `features_lib.Audio` so that if any other dataset like video or text is passed it can print error

Reply:
I tried this - feature_lib.Audio was returning None for both the tested datasets. Since that instantiation check was not working - I had to resort to this implementation. [][Here](https://github.com/tensorflow/datasets/pull/1683/commits/cfb259b8ad3b828935529f341cd9b1e2a86fd103)) is the commit with that code - Perhaps you could point out the mistake?

Reply:
Don't know why its not working , dataset you are trying for audio_keys have how many number of channels, Audio feature supports only 1-D values [see](https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/core/features/audio_feature.py#L47)

Reply:
Just rechecked - I think I was missing something trying it earlier - the instantiation check is working (at least with the feature_lib.Audio) so I will add that. My bad.  
Thanks for the code style check reminder - have done that as well and will make the required changes! :+1: